lerna.json

```diff
@@ -1,6 +1,4 @@
 {
   "npmClient": "yarn",
-  "useWorkspaces": true,
```

Comment:
`useWorkspaces` is defaulted now and removed in v7. [Changelog](https://github.com/lerna/lerna/releases/tag/7.0.0)